views/trials-details.html

```diff
@@ -78,5 +78,23 @@
       </dd>
     {% endfor %}
   </dl>
+
+  <h2>Publications</h2>
+  <dl>
+    {% for publication in trial.publications %}
+      <dt>
+        <a href="/publications/{{ publication.attributes.id }}">
+          {{ publication.attributes.title }}
+       </a>
+      </dt>
+      <dd>
+        Last updated on
+        <time datetime="{{ publication.attributes.updated_at }}">
+          {{ publication.attributes.updated_at | formatDate }}
```

Comment:
No need for the last updated date. Just show the publication title here, probably using a `<ul>` instead of a `<dl>`